src/scripts/githubAPI.js

```diff
@@ -85,6 +85,11 @@ query summaryQuery($from: DateTime!, $username: [String!]) {
   }
```

Comment:
_Not from this PR I guess. But could you change this_

1. ![image](https://user-images.githubusercontent.com/28642011/87660786-3eff9c80-c77d-11ea-99bd-4976954567ca.png)

The `NOTES` seem to be missing from the path.

current  --> `HOME/  /MY-TODO`
expected --> `HOME/ NOTES / MY-TODO`

2. Also, clicking on To-do seem to do nothing as of now (the screen remains the same as Standup Notes)

Reply:
![Screenshot 2020-07-16 at 4 06 21 PM](https://user-images.githubusercontent.com/11258286/87661477-4d01ed00-c77e-11ea-9874-c2cd75df6d6d.png)



This PR Fixes this already

Reply:
I think, it's because of missing .env. Coz it wasn't working on vercel preview